src/components/Radio.tsx

```diff
@@ -1,35 +1,38 @@
+import React from 'react';
 import { PropsWithChildren } from "react";
 import { colors } from "../theme";
 import styled from "styled-components";
 import { InputHTMLAttributes } from "react";
+import {useTooltipTriggerState} from 'react-stately';
+import {useTooltipTrigger} from 'react-aria';
+import { CustomTooltip } from './Tooltip';
 
-const StyledLabel = styled.label<{disabled?: boolean}>`
+export const StyledLabel = styled.label<{isDisabled?: boolean}>`
   font-size: 1.4rem;
   display: flex;
   align-items: center;
-  color: ${(props => props.disabled ? colors.palette.neutral : 'inherit')}
+  color: ${(props => props.isDisabled ? colors.palette.pale : 'inherit')};
+  width: 100%;
+  position: relative;
 `;
 
 // https://moderncss.dev/pure-css-custom-styled-radio-buttons/
-const StyledInput = styled.input`
+export const StyledInput = styled.input<{isDisabled?: boolean}>`
   appearance: none;
   /* For iOS < 15 to remove gradient background */
   background-color: ${colors.palette.white};
+  opacity: ${(props => props.isDisabled ? '40%' : '100%')};
```

Comment:
For the disabled state, the opacity of the border looks too light to me:
<img width="147" alt="image" src="https://github.com/openstax/ui-components/assets/34174/ef4ffdd3-bcb5-4714-8a4c-1360c8713d7c">
Maybe bumping up the value will make it easier to tell there's a control there?